app/proto/auth.proto

```diff
@@ -243,3 +247,12 @@ message ContributorForm {
   repeated string contribute_ways = 5;
   string expertise = 6;
 }
+
+message UnsubscribeReq {
+  bytes payload = 1;
+  bytes sig = 2;
```

Comment:
Please describe if these should be base64url encoded or raw bytes.

Reply:
base64 would be `string` instead of `bytes`, but I've added a comment as well!